models/demos/falcon7b_common/tests/test_utils.py

```diff
@@ -87,99 +97,146 @@ def get_rand_falcon_inputs(
         assert q_len % 32 == 0, "For prefill, seq_len must be multiple of 32!"
         assert kv_cache_len == 0, "For prefill, no kv_cache is passed in!"
 
+        # Generate attention input and mask
         if generate_attention_inputs:
             attention_input = (torch.rand(global_batch, q_len, configuration.hidden_size) * 2) - 1
             attention_mask_bool = torch.ones(global_batch, 1, q_len, kv_len, dtype=bool).triu(diagonal=1)
-        layer_past = None
+            tt_attention_input = tt_from_torch(
```

Comment:
I think tt_from_torch is redundant. You can use if else directly in the code instead of using wrapper for it. I believe it just adds unnecessary complexity.

Reply:
I added that because the if else would be then be required in many falcon7b files and it's something that can easily be forgotten when making future changes to Falcon7b. I believe this can help safeguard against that.